docs/static-run.md

```diff
@@ -20,6 +21,7 @@ To run all standard tests, from the controller:
 
 ```console
 export TESTHOST="${PREFIX}suma3pg.tf.local"
+export PROXY="None"
```

Comment:
can we use 'NOT_SET' or something else then 'None' because in python and ruby lead to misbehaviour with the nil object. :rocket:

Reply:
I would prefer over all a boolean var false

Reply:
"Non" is what Salt answers if a grain is not set. That's why I am using this.

We have a kind of boolean: $proxy.nil?